vic/drivers/image/Makefile

```diff
@@ -90,11 +90,10 @@ INCLUDES = -I ${DRIVERPATH}/include \
 		   -I ${SHAREDPATH}/include \
 		   -I ${SHAREDIMAGEPATH}/include \
 		   -I ${NETCDFPATH}/include \
-		   -I ${MPIPATH}/include
 
 # Uncomment to include debugging information
 CFLAGS  =  ${INCLUDES} -ggdb -O0 -Wall -Wextra -std=c99 -DLOG_LVL=$(LOG_LVL)
-LIBRARY = -lm -L${NETCDFPATH}/lib -lnetcdf -L${MPIPATH}/lib -lmpi
+LIBRARY = -lm -L${NETCDFPATH}/lib -lnetcdf
```

Comment:
A note on this.  It turns out we don't need to explicitly link to the mpi libraries. This is done via `mpicc` which is basically an alias for the compiler flags and links that need to happen.  For example, 

```bash
$ mpicc -show
/usr/bin/clang -pipe -arch x86_64 -Wl,-headerpad_max_install_names -arch x86_64 -Wl,-flat_namespace -Wl,-commons,use_dylibs -I/opt/local/include/mpich-mp -L/opt/local/lib/mpich-mp -lmpi -lpmpi
```